test/test_common/environment.cc

```diff
@@ -126,8 +126,10 @@ std::string TestEnvironment::substitute(const std::string& str,
   switch (version) {
   case Network::Address::IpVersion::v4:
     out_json_string = std::regex_replace(out_json_string, lookup_family_regex, "v4_only");
+    FALLTHRU;
```

Comment:
I don't think this actually makes sense given the context? Should probably be break here and below?